core/base/inc/TDirectory.h

```diff
@@ -266,6 +266,10 @@ can be replaced with the simpler and exception safe:
       {
          return WriteObjectAny(obj, TClass::GetClass<T>(), name, option, bufsize);
       }
+   inline Int_t WriteObject(const TObject* obj, const char* name, Option_t *option="", Int_t bufsize=0)
+   {
+      return WriteTObject(obj, name, option, bufsize);
+   }
```

Comment:
White space to match previous function's indentation.
```suggestion
      {
         return WriteTObject(obj, name, option, bufsize);
      }
```